content/learn/08.contributions/03.arduino-creating-library-guide/arduino-creating-library-guide.md

```diff
@@ -243,6 +259,7 @@ If you tried the new sketch, you probably noticed that nothing from our library
 
 ```arduino
 Morse	KEYWORD1
+begin KEYWORD2
```

Comment:
```suggestion
begin	KEYWORD2
```
`keywords.txt` uses a tab-separated data format, and doesn't support space separators:

https://arduino.github.io/arduino-cli/dev/library-specification/#keywordstxt-format

Reply:
Ugh... this gets me every time. My editor is set to expand tabs for python projects. I thought I checked this before I commited, but clearly not.  Thank you! 🙇